packages/reactive-element/package.json

```diff
@@ -381,7 +384,8 @@
     "/node/"
   ],
   "dependencies": {
-    "@lit-labs/ssr-dom-shim": "^1.1.2"
+    "@lit-labs/ssr-dom-shim": "^1.1.2",
+    "lit-html": "^3.1.0"
```

Comment:
You're right. Technically the field `_$litStatic$` is unminified and the brand `Symbol.for('')` is recreateable, but it would be more proper to add the dependency and import the type.